apps/www/src/styles/globals.css

```
@@ -10,6 +10,9 @@
 
 @source "./src/**/*.{js,ts,jsx,tsx}";
 @source "../../node_modules/@chia/ui/src/**/*.{js,ts,jsx,tsx}";
+@source "../../node_modules/@chia/agent-elements/src/**/*.{js,ts,jsx,tsx}";
+@source "../../node_modules/@chia/agent-elements/node_modules/streamdown/dist/*.js";
+@source "../../node_modules/@chia/agent-elements/node_modules/@streamdown/cjk/dist/*.js";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)